packages/bigtable/test/table.js

```diff
@@ -124,44 +120,35 @@ describe('Bigtable/Table', function() {
         return FAKE_TABLE_NAME;
       });
 
-      var table = new Table(BIGTABLE, TABLE_ID);
+      var table = new Table(INSTANCE, TABLE_ID);
       var config = table.calledWith_[0];
 
       assert(table instanceof FakeGrpcServiceObject);
-      assert.strictEqual(config.parent, BIGTABLE);
+      assert.strictEqual(config.parent, INSTANCE);
       assert.strictEqual(config.id, FAKE_TABLE_NAME);
 
       assert.deepEqual(config.methods, {
         create: true,
         delete: {
           protoOpts: {
-            service: 'BigtableTableService',
+            service: 'BigtableTableAdmin',
             method: 'deleteTable'
           },
           reqOpts: {
             name: FAKE_TABLE_NAME
           }
         },
         exists: true,
-        get: true,
-        getMetadata: {
-          protoOpts: {
-            service: 'BigtableTableService',
-            method: 'getTable'
-          },
-          reqOpts: {
-            name: FAKE_TABLE_NAME
-          }
-        }
+        get: true
       });
 
-      assert(Table.formatName_.calledWith(BIGTABLE.clusterName, TABLE_ID));
+      assert(Table.formatName_.calledWith(INSTANCE.id, TABLE_ID));
     });
 
-    it('should use Bigtable#createTable to create the table', function(done) {
+    it('should use Instance#createTable to create the table', function(done) {
```

Comment:
I obviously let this slip the first time around reviewing the Bigtable PR, but can you catch the places where we call `.createMethod` and use the method that's passed to `config.createMethod` instead?